spk/inotify-tools/src/service-setup.sh

```diff
@@ -0,0 +1,27 @@
+
+BIN_COMMANDS="inotifywait inotifywatch"
+LIBRARY="libinotifytools.so"
+LIBRARY_MAIN_VERSION="0"
+LIBRARY_SUB_VERSION="4.1"
+LIBRARY_WITH_VERSION="${LIBRARY}.${LIBRARY_MAIN_VERSION}.${LIBRARY_SUB_VERSION}"
```

Comment:
I do not understand what these variables are expected to operate here. From my point of view, should be removed.

Reply:
This is resolved by @cyc1ingsir and the whole service-setup.sh is removed by usage of generic symlink creation.